chroot-bin/croutonwheel

```diff
@@ -64,8 +82,12 @@ flock 3
 {
     croutonxi2event &
     xi="$!"
-    aura="`croutonwmtools list ni | awk '/aura_root/ {print $NF}'`"
-    xev -id "$aura" &
+
+    if [ ! -L /etc/crouton/cmt ]; then
```

Comment:
Probably should do the xmethod/link test once at the top and store the result in a variable.